ambari/src/main/java/io/brooklyn/ambari/AmbariCluster.java

```diff
@@ -145,6 +145,20 @@
 
     AttributeSensor<Boolean> CLUSTER_SERVICES_INSTALLED = Sensors.newBooleanSensor("ambari.cluster.servicesInstalled");
 
+    @SetFromFlag("ambariAlertNotifications")
+    ConfigKey<Map<String, Object>> AMBARI_ALERT_NOTIFICATIONS =
+            new MapConfigKey(Map.class, "ambari.alert.notifications", "Map of maps");
+
+    String AMBARI_ALERTS_CONFIG_PREFIX = "ambari.alerts.notification.";
```

Comment:
I wonder whether it is overkill to put prefix for all `AMBARI_ALERT_*` settings.
Also you have defined `ambari.alert.notifications` previously?